src/main/resources/view/MainWindow.fxml

```diff
@@ -1,59 +1,85 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
+<?import java.lang.String?>
 <?import java.net.URL?>
```

Comment:
Add space below.

Reply:
agree with this, will make changes and update the pr

Reply:
Nice